analyticsclient/module.py

```diff
@@ -29,22 +28,6 @@ def answer_distribution(self, data_format=DF.JSON):
 
         return self.client.get(path, data_format=data_format)
 
-    def submission_counts(self, module_ids, data_format=DF.JSON):
```

Comment:
There should be nothing using this endpoint. It was added for the answer distribution work.